src/Fields/KeyValueField.php

```diff
@@ -81,13 +81,13 @@ public function __construct($name, $title = null, $sourceKeys = [], $sourceValue
     public function Field($properties = [])
     {
         if (Controller::curr() instanceof ContentController) {
-            Requirements::javascript('silverstripe/admin: thirdparty/jquery/jquery.js');
+            Requirements::javascript('https://code.jquery.com/jquery-3.6.3.min.js');
```

Comment:
```suggestion
            Requirements::javascript('//code.jquery.com/jquery-latest.min.js');
```
This is unlikely to get updated manually whenever jquery patches are released, so we should just use the latest one.

Reply:
I'm also unsure if `Requirements::javascript()` accepts `//` or not.... if it doesn't, https will do.